src/qminer/qminer_aggr.h

```diff
@@ -1771,11 +1774,174 @@ class TRecFilterAggr : public TStreamAggr {
     TStr Type() const { return GetType(); }
 };
 
+typedef enum {
```

Comment:
This should all move out of qminer to glib/mine, preferably to its own namespace?